guides/source/active_record_basics.md

```diff
@@ -162,14 +175,42 @@ CREATE TABLE products (
 );
 ```
 
-The schema above declares a table with two columns: `id` and `name`. Each row of
-this table represents a certain product with these two parameters. Thus, you
-would be able to write code like the following:
+Database tables in Rails are typically created using [Active Record Migrations](#migrations) and not raw SQL. A migration for the `products` table above can be generated like this:
+
+```bash
+$ rails generate migration CreateProducts name:string
+```
+
+and results in this:
 
 ```ruby
+# Note: the `id` column, as the primary key, is autometically created by convension. As are `created_at` and `updated_at` columns.
+
+# /db/migrate/20240220143807_create_products.rb
+class CreateProducts < ActiveRecord::Migration
```

Comment:
This is a totally optional suggestion and not necessary - but I like the idea of using the same example throughout the guide if we can.  For example, we use a "Book" model below and it would be cool to first see how a books table is set up and used, rather than having products table set up and then books be queried.

Again, this is just my preference, I'm curious to hear your thoughts?

Reply:
This is from the original guide. I created all the Book examples. Will match.